src/pages/webhooks/use-cases/code-samples/order-place-custom-attributes-validation-admin.md

```diff
@@ -0,0 +1,40 @@
+```yaml
+Hook Settings
+
+Webhook method: observer.sales_order_place_before
+Webhook type: before
+Batch name: order_validation
+Batch order: 100
+Hook name: order_custom_attributes_validation
+Hook priority: 100
+URL: <Host>/validate-order-attributes
+Timeout: 5000
```

Comment:
We can increase the timeout in the example. As real validation action can make some external calls. We can remove it or put a larger value (30s ?)

Reply:
And the same for other examples